e2e/teams.loggedin.spec.ts

```diff
@@ -1,29 +1,35 @@
 import { Page, expect, test } from '@playwright/test'
 import { v4 as uuidv4 } from 'uuid'
-import { ADMIN_STORAGE_STATE, USER_STORAGE_STATE } from '../playwright.config'
+import {
+  USER_STORAGE_STATE,
+  USER_VALIDATED_STORAGE_STATE,
+} from '../playwright.config'
 import { getLocalStorage } from './helpers/localStorage'
 import {
   deleteTeamInvitationMessage,
   deleteTeamInviteAcceptanceMessage,
 } from './helpers/messages'
 import {
-  USER_NAME_LOCALSTORAGE_KEY,
   getAccessTokenFromCookie,
-  getAdminUserCredentials,
+  getAdminPAT,
   getUserIdFromLocalStorage,
   goToDashboard,
 } from './helpers/testUser'
+import {
+  USER_NAME_LOCALSTORAGE_KEY,
+  USER_VALIDATED_NAME_LOCALSTORAGE_KEY,
+} from './helpers/userConfig'
 
 const TEAM_NAME = 'swc-e2e-team-' + uuidv4()
-const INVITATION_MESSAGE = 'swc e2e test team invitation' + uuidv4()
+const INVITATION_MESSAGE = 'swc-e2e-invite'
 
-let adminPage: Page
+let validatedUserPage: Page
 let userPage: Page
 
 test.describe('Teams', () => {
   test.beforeAll(async ({ browser }) => {
-    adminPage = await browser.newPage({
-      storageState: ADMIN_STORAGE_STATE,
+    validatedUserPage = await browser.newPage({
+      storageState: USER_VALIDATED_STORAGE_STATE,
```

Comment:
Use validated user instead of admin for teams test, which requires multiple users